### DISCUSSION TITLE
GETTING ERROR WHILE INSTALLING AUTBAHN

### DISCUSSION THREAD

**Discussion by MooneDrJune** [TechfaneTechnologies/QtsApp#3] [Q&A] [answered]
### Question: `getting error while installing autbahn_rce`
![image](https://user-images.githubusercontent.com/96371033/179363640-5b996e57-78b0-4732-a2d6-01cfee44b6af.png)

### Answer: `pip install autobahn[twisted,asyncio,encryption,serialization,xbr]`

**Comment by MooneDrJune** [comment, marked as answer]
`pip install -U autobahn numpy pandas python-dotenv requests six Twisted websocket_client xlwings autobahn[twisted,asyncio,encryption,serialization,xbr] `

**Comment by VadivelanTEST** [comment]
Traceback (most recent call last):
  File "C:\wamp64\www\zip\hlo\QtsApp\example.py", line 32, in <module>
    EnchantQtsApp(
  File "C:\Users\Eagleminds\AppData\Local\Programs\Python\Python311\Lib\site-packages\qtsapp\QtsApp.py", line 158, in EnchantQtsApp
    del qtsapp
        ^^^^^^
UnboundLocalError: cannot access local variable 'qtsapp' where it is not associated with a value
